File: the-rhinestone-cowboy-~-count-the-dollars-in-his-boots/the-rhinestone-cowboy-~-count-the-dollars-in-his-boots.js
function cowboysDollars(boots) {
var botlef = boots[0].substring(0, boots[0].indexOf("&")).match(/\[\(1\)\]/g)
var botrgt = boots[1].substring(0, boots[0].indexOf("&")).match(/\[\(1\)\]/g)
return "This Rhinestone Cowboy has "+[botrgt==null? "0":botrgt.length]+ " dollar bills in his right boot and "+[botlef==null? "0":botlef.length]+" in the left"
}

File: the-rhinestone-cowboy-~-count-the-dollars-in-his-boots/the-rhinestone-cowboy-~-count-the-dollars-in-his-boots.py
import re
def cowboys_dollars(boots):
    botlef = re.findall(r"\[\(1\)\]",boots[0][0:boots[0].index("&")])
    botrgt = re.findall(r"\[\(1\)\]",boots[1][0:boots[1].index("&")])
    return "This Rhinestone Cowboy has "+str(len(botrgt))+ " dollar bills in his right boot and "+str(len(botlef))+" in the left"